dawn/main/ansible/ansible.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package main;
+
+message GamepadValueList {
+    repeated sint32 val = 1;
+}
+
+message Gamepad {
+    GamepadValueList axes = 1;
+    GamepadValueList buttons = 2;
+}
+
+message GamepadData {
+    repeated Gamepad gamepad = 1;
+}
+
+message ConsoleOutput {
+    string output = 1;
+}
+
+message AnsibleMessage {
+    string type = 1;
+    bytes data = 2;
```

Comment:
I notice you're storing bytes here, and then parsing them as proto again. Have you tried using proto3's "Any" type instead? (I'll need to check, but I expect the "Any" type to be more space-efficient while also offering a better API)